exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/FileSelection.java

```diff
@@ -194,7 +193,7 @@ private static String commonPathForFiles(final List<String> files) {
     int shortest = Integer.MAX_VALUE;
     for (int i = 0; i < total; i++) {
       final Path path = new Path(files.get(i));
-      folders[i] = Path.getPathWithoutSchemeAndAuthority(path).toString().split(PATH_SEPARATOR);
+      folders[i] = Path.getPathWithoutSchemeAndAuthority(path).toString().split(FileUtils.PATH_SEPARATOR_QUOTED);
```

Comment:
I don't think that `PATH_SEPARATOR_QUOTED` should be used there since `org.apache.hadoop.fs.Path` was used which should use `/` separator independently on the system.